pkg/output/http/config.go

```diff
@@ -12,6 +12,7 @@ type Config struct {
 	BatchTimeout       time.Duration     `yaml:"batch_timeout" default:"5s"`
 	ExportTimeout      time.Duration     `yaml:"export_timeout" default:"30s"`
 	MaxExportBatchSize int               `yaml:"max_export_batch_size" default:"512"`
+	GzipCompression    bool              `yaml:"gzip_compression" default:"false"`
```

Comment:
non blocking: should this be enum with the strategy of compression? (none, gzip, ...)

```
Compression    CompressionStrategy              `yaml:"compression" default:"none"`
```

Reply:
Yeah SGTM

Reply:
Updated

Reply:
:sendit: